packages/playground/src/layouts/RepoLayout.tsx

```diff
@@ -7,10 +7,10 @@ const RepoLayout: React.FC = () => {
 
   return (
     <div>
-      <header style={{ background: '#333', padding: '20px', height: '56px' }}>
+      <header className="p-5 h-16 bg-black">
```

Comment:
let's not use absolute colors like `black`. use semantic colors instead, like `bg-background`

Reply:
Solved from bg-black to bg-background